dom-manipulation/main.js

```diff
@@ -0,0 +1,20 @@
+let clickCounter = 0;
+const $hotBtn = document.querySelector('.hot-button');
+const $clkCount = document.querySelector('.click-count');
+$hotBtn.addEventListener('click', () => {
+  clickCounter++
+  document.querySelector(".click-count").textContent = clickCounter;
```

Comment:
You already have a variable that references your `click-count` element, so just use that variable name instead of querying for the DOM again